pwaCustomButton.js

```diff
@@ -0,0 +1,30 @@
+if (typeof window !== 'undefined') {
+  window.addEventListener('beforeinstallprompt', (e) => {
+    const pwaBtn = document.querySelector('#pwa-button');
+
+    // Prevent Chrome 67 and earlier from automatically showing the prompt
+    e.preventDefault();
+    window.deferredPrompt = e;
+
+    // Display A2HS button if app not installed
+    pwaBtn.removeAttribute('hidden');
+
+    pwaBtn.addEventListener('click', () => {
+      // hide our user interface that shows our A2HS button
+      pwaBtn.hidden = true;
```

Comment:
Let's be consistent.  Either do it via JS getters/setters like this (which is probably right), or use DOM attributes like line 10.  I would change line 10 to `pwaBtn.hidden = false;`

Reply:
Yes right thanks, fixed